femmt/examples/basic_transformer.py

```diff
@@ -81,17 +81,27 @@ def example_thermal_simulation(show_visual_outputs: bool = True, flag_insulation
         os.mkdir(working_directory)
 
     # 1. chose simulation type
-    geo = fmt.MagneticComponent(component_type=fmt.ComponentType.Transformer, working_directory=working_directory,
+    geo = fmt.MagneticComponent(simulation_type=fmt.SimulationType.FreqDomain, component_type=fmt.ComponentType.Transformer, working_directory=working_directory,
```

Comment:
i do not agree with changing this file in general.
There should be a new file called basic_transformer_time_domain.py to show the time domain simulation for the transformer

Reply:
ok, found the file some lines down.
Nevertheless, this file should not be changed.